org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractAutoEditTest.java

```diff
@@ -50,6 +51,15 @@ public void tearDown() throws Exception {
 		files.clear();
 		closeEditors();
 		super.tearDown();
+		waitForEventProcessing();
+	}
+	
+	/**
+	 * @since 2.23
+	 */
+	protected void waitForEventProcessing() {
```

Comment:
Should this be called from `closeEditors`?

Reply:
good question. or from the baseclass

Reply:
I've a preference towards pushing `org.eclipse.xtext.ui.testing.AbstractHighlightingTest.waitForEventProcessing()` into the `AbstractWorkbenchTest` and calling it from its tearDown.